egs/babel/s5d/conf/mfcc_hires_bp.conf

```diff
@@ -0,0 +1,10 @@
+# config for high-resolution MFCC features, intended for neural network training.
+# Note: we keep all cepstra, so it has the same info as filterbank features,
+# but MFCC is more easily compressible (because less correlated) which is why
+# we prefer this method.
+--use-energy=false   # use average of log energy, not energy.
+--sample-frequency=8000 #  Switchboard is sampled at 8kHz
```

Comment:
Yes, this is useful when there is different types of data like microphone data mixed with telephone as is the case with Babel and maybe Aspire.